docs/wegue-configuration.md

```diff
@@ -127,6 +128,40 @@ Below is an example for such a configuration object:
   }
 ```
 
+### viewAnimation
+
+Map views can be animated, to zoom or pan from the current to a target location, typically after a user action takes place. This can be configured by the property `viewAnimation` in the main Wegue configuration. Per default animations are disabled.
```

Comment:
```suggestion
Map views can be animated, to zoom or pan from the current to a target location, typically after an user action takes place. This can be configured by the property `viewAnimation` in the main Wegue configuration. Per default animations are disabled.
```

Reply:
Hi Jakob. I don't think this should be 'an', because what matters is whether the following word starts with a vowel **phonetically**. See https://www.grammar.com/a-vs-an-when-to-use/

Reply:
Okay. Does not matter :-)